PostgreSQL/README.md

```diff
@@ -0,0 +1,8 @@
+# PostgreSQL
+
+PostgreSQL marketplace status of the following
+
+| Type | File | Status |
+|--|--|--|
+| Connector | [connector.js](./connector.js) | ✅ |
+| Prepare | [node.prepare.js](./node.prepare.js) | ✅ |
```

Comment:
The node name would be SQL Builder and not prepare. That changed based on the discussion we had with @jugnuagrawal

Reply:
Fixed.